configs/peer/genesis.json

```diff
@@ -7,7 +7,7 @@
             "object": {
               "Raw": {
                 "Identifiable": {
-                  "Domain": {
+                  "NewDomain": {
```

Comment:
For consistency? This needs to both have the `API_CHANGES` tag and we should notify everyone.

Reply:
not just consistency, but also to prevent defining inner structure of `Domain` AOT. E.g., `NewDomain` prevents the user to define accounts on the domain AOT, accounts can only be added through separate instructions

Reply:
I like it. Keep up the good work.